pkg/agent/openflow/client.go

```diff
@@ -708,6 +708,7 @@ func (c *client) SendTraceflowPacket(
 	case 6:
 		packetOutBuilder = packetOutBuilder.SetIPProtocol(binding.ProtocolTCP)
 		if TCPSrcPort == 0 {
+			// #nosec G404: Use of weak random number generator (math/rand instead of crypto/rand)
```

Comment:
```suggestion
			// #nosec G404: random number generator not used for security purposes
```

I think the comment should explain why we are silencing the warning, not what the warning is 

same in other places